main.py

```diff
@@ -22,13 +25,18 @@
         sys.exit(1)
 
     transactions = TransactionCollection()
+    payments = DividendCollection()
 
     if args.transactions_trading212:
         Trading212TransactionsImporter.import_transactions(transactions, args.transactions_trading212)
+        #Trading212DividendsImporter.import_dividends(transactions, args.dividends_trading212)
```

Comment:
Let's keep this, and pass it the `args.transactions_trading212` instead of the `args.dividends_trading212` (which we can remove).